### PR TITLE
use alibuild-generate-module

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

**Comment by awegrzyn** [comment]
I think you need to add ` alibuild-recipe-tools` to `build_requires`

**Comment by Barthelemy** [comment]
@awegrzyn thank you

File: abseil.sh [hunk 1]
```diff
@@ -20,20 +21,13 @@ cmake $SOURCEDIR                        \
...
-set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
-module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
-# Dependencies
-module load BASE/1.0                                                          \\
-            ${GCC_TOOLCHAIN_REVISION:+GCC-Toolchain/$GCC_TOOLCHAIN_VERSION-$GCC_TOOLCHAIN_REVISION}
+alibuild-generate-module > "$MODULEFILE"
+cat >> "$MODULEFILE" <<EoF
+
 # Our environment
 set ABSEIL_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
```
Comment (ktf) [inline abseil.sh:32]: Note that we do not really need those, because abseil does not have any bin folder...

File: mpfr.sh [hunk 2]
```diff
@@ -26,16 +27,9 @@ rm -f $INSTALLROOT/lib/*.la
...
-}
-set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
-module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
-# Dependencies
-module load BASE/1.0
+alibuild-generate-module > "$MODULEFILE"
+cat >> "$MODULEFILE" <<EoF
+
 # Our environment
 set MPFR_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
```
Comment (ktf) [inline mpfr.sh:34]: Same here...

**Comment by ktf** [comment]
I think this is fine for now. In retrospect we do not need the modulefile at all, since IIRC I removed the bit from the script which was mandating the need for one in order to be published...

**Comment by Barthelemy** [comment]
ok, as we need a new release of these packages, I propose to keep it as it is as it does not hurt.

**Pull request merged**